TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj

```diff
@@ -21,5 +21,11 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <DotNetCliToolsReference Include="dotnet-portable">
```

Comment:
Where does this tool come from?  Is it built in our repo?  When we restore it in the tests, I don't see a special package source being defined.